sign_oca/models/sign_oca_request.py

```diff
@@ -265,6 +265,35 @@ def action_send(self, sign_now=False, message=""):
                 email_layout_xmlid="mail.mail_notification_light",
             )
 
+    def action_send_signed_request(self):
+        self.ensure_one()
+        if self.state != "signed":
+            return
+        for signer in self.signer_ids:
+            attachments = (
+                self.env["ir.attachment"]
+                .sudo()
+                .search(
+                    [
+                        ("res_model", "=", "sign.oca.request"),
+                        ("res_id", "=", self.id),
+                        ("res_field", "=", "data"),
+                    ]
+                )
+            )
+            self.env["mail.thread"].message_notify(
```

Comment:
Can you explain in a code comment why it is not done on the sign request record?

Reply:
What do you mean it is not done on the sign request record? The code is in the sign request model but we notify all the signers.
What should I exactly clarify?

Reply:
Okay, done!